.github/workflows/main.yml

```diff
@@ -7,34 +7,40 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 
+env:
+  PNPM_VERSION: 9.1.1
+
 jobs:
   build:
-    name: Build, lint, and test on Node ${{ matrix.node }} and ${{ matrix.os }}
-
-    runs-on: ${{ matrix.os }}
+    name: Build, lint, and test on Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         node: ["20.x"]
```

Comment:
This matrix config should be removed, it's actually unused, for now i just removed the os.

Reply:
Yep, I agree. This is probably a holdover from https://github.com/achievements-app/psn-api. This library is loosely based on psn-api.

Reply:
Ok, let's tackle that in other pr.

Reply:
Sorry, what I mean to say is I agree with your change here 👍